assets/base.css

```diff
@@ -1132,6 +1152,16 @@ summary::-webkit-details-marker {
   .slider--tablet.grid--peek.grid--2-col-tablet .grid__item {
     width: calc(50% - var(--grid-desktop-horizontal-spacing) - 3rem);
   }
+}
+
+@media screen and (max-width: 989px) {
+  .slider--tablet.grid--peek.grid--3-col-tablet-down .grid__item {
+    width: calc(33.33% - var(--grid-desktop-horizontal-spacing) - 3rem);
+  }
+
+  .slider--tablet.grid--peek.grid--2-col-tablet-down .grid__item {
+    width: calc(50% - var(--grid-desktop-horizontal-spacing) - 3rem);
+  }
```

Comment:
Same concern as above. I believe this ends up using `--grid-desktop-horizontal-spacing` for mobile as well as it would win in specificity over `.grid--2-col-tablet-down.grid--peek .grid__item` on line 1178

Reply:
Good catch, Ken! I updated the class specifity